### PR TITLE
feat-cmd-SysProcAttr: RunOptions add SysProcAttr parameter Let exec.C…

### PR DESCRIPTION
**Pull request by hongcha98**
use #https://github.com/playwright-community/playwright-go/issues/376

### CODE REVIEW

File: run.go [hunk 1]
```diff
@@ -193,6 +197,9 @@ func (d *PlaywrightDriver) DownloadDriver() error {
 
 func (d *PlaywrightDriver) run() (*connection, error) {
 	cmd := exec.Command(d.DriverBinaryLocation, "run-driver")
+	if d.options != nil {
```
Comment (mxschmitt) [inline run.go:200]: I feel like this should be a default. So the user does not have to pass it over. We do it in the ports too:
- Python: https://github.com/microsoft/playwright-python/blob/59369fe126f49c10597d5c9099840bdc8ccdcf15/playwright/_impl/_transport.py#L121
- .NET: https://github.com/microsoft/playwright-dotnet/blob/eade3f5894752dc64cc1c242a37f03b39355c625/src/Playwright/Transport/StdIOTransport.cs#L122
- .NET: https://github.com/microsoft/playwright-dotnet/blob/eade3f5894752dc64cc1c242a37f03b39355c625/src/Playwright/Program.cs#L55-L57

Comment (hongcha98) [reply run.go:200]: There is no such processing now. Will you consider adding it by default in the future?

Comment (hongcha98) [reply run.go]: I added a defaultSysProcAttr, what do you think?

**Review by mxschmitt — APPROVED** [review]

**Review by canstand — COMMENTED** [review]
hi @hongcha98, could you add same logic to `cmd/playwright/main.go`?

**Comment by hongcha98** [comment]
> hi @hongcha98, could you add same logic to `cmd/playwright/main.go`?

The initial idea was to pass parameters and I wrote this [example](https://github.com/playwright-community/playwright-go/pull/377/commits/07b8de08100ee11a7b607e04b2c8195a7cc32464). Finally, defaultSysProcAttr is used to implement it, so it is not needed.

**Pull request merged**